cloud/pkg/cloudstream/tunnelserver.go

```diff
@@ -169,3 +184,30 @@ func (s *TunnelServer) Start() {
 		return
 	}
 }
+
+func (s *TunnelServer) updateNodeKubeletEndpoint(nodeName string) {
+	if err := wait.PollImmediate(retrySleepTime, nodeStatusUpdateRetry, func() (bool, error) {
```

Comment:
```suggestion
	if err := wait.Poll(retrySleepTime, nodeStatusUpdateRetry, func() (bool, error) {
```
Let's use `wait.Poll` here, PollImmediate will exit when err occur, here we need to retry.

Reply:
Thanks for the reply. 
I test in my machine, if err happens retry will execute.
`wait.PollImmediate` will run the func immediately, and  `wait.Poll` will wait for the interval firstly.

Reply:
Okey, sounds good, and so we need to return `false,nil` as I comment above. Or the retry will exit directly.

Reply:
Thanks for the reply, I have changed the `return false, err` to `return false, nil`.